### PR TITLE
create makefile

### PR DESCRIPTION
**Pull request by ntatrishvili**
creating #13

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -* MakeFile -*-
+
+mc_stdin = /run/minecraft.stdin
+logss = ~/minecraft/app/logs/latest.log
```
Comment (samerbahri98) [inline makefile]: are you sure it's that's where the logs are in the server?

File: makefile [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
```
Comment (samerbahri98) [inline makefile]: where is the start?

File: makefile
Comment (samerbahri98) [inline makefile:1]: all the targets are phony targets, none of them should creates a file!

**Review by samerbahri98 — CHANGES_REQUESTED** [review]

File: makefile [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# -* MakeFile -*-
+
+mc_stdin = /run/minecraft.stdin
+_logs = /app/logs/latest.log
+_minecraft = minecraft
+
```
Comment (samerbahri98) [inline makefile:6]: the start target is missing

File: iac/cloudinit/cloud-init.yml [hunk 4]
```diff
@@ -67,4 +67,5 @@ runcmd:
   - ["systemctl", "daemon-reload" ]
   - ["systemctl", "enable", "minecraft"]
   - ["systemctl", "start", "--no-block", "minecraft"]
+  - ["cd minecraft && make"]
```
Comment (samerbahri98) [inline iac/cloudinit/cloud-init.yml:70]: this should not be like this.
 - the directory is wrong
 - the make has no default target (which means the default is stopping the server)

**Review by samerbahri98 — CHANGES_REQUESTED** [review]
almost there,

File: iac/cloudinit/cloud-init.yml (hunk 4 above)
Comment (ntatrishvili) [reply iac/cloudinit/cloud-init.yml:70]: I tried the directory in my computer and it only worked like thissss

**Pull request merged**